lib/toc.rb

```diff
@@ -77,11 +67,8 @@ def guide_name
     end
 
     def chapter_name
-      if current_chapter
-        return current_chapter.title
-      else
-        return ""
-      end
+      return "" unless  current_chapter
```

Comment:
We could probably nuke the returns in the original, but I don't think this adds readability